CONTRIBUTING.md

```diff
@@ -64,4 +65,24 @@
     git add -f *.min.js*
     git rm .gitignore
     git commit -m 'Add build artifacts from '`cat .git/refs/remotes/web-animations-next/master`
-    git push web-animations-js HEAD:refs/heads/master
\ No newline at end of file
+    git push web-animations-js HEAD:refs/heads/master
+
+## Testing architecture
+
+This is an overview of what happens when `grunt test` is run.
+
+1. Polyfill tests written in mocha and chai are run.
+1.1 grunt creates a karma config with mocha and chai adapters.
```

Comment:
This markup doesn't seem to be correct; the preview is not creating the sublists correctly.